### PR TITLE
Converted chemmaster to TGUI

### PR DESCRIPTION
**Pull request by PeterArbeitsfrei**
Does what it says in title: Converts the chemmaster to using tgui.

I would like to get some feedback on the Look and how bad my code is.

https://imgur.com/a/fyBiT

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Niiice

**Comment by ChangelingRain** [comment]
The eject button should be in more of a warning color tbh

**Comment by Iamgoofball** [comment]
sign me the FUCK up 👌👀👌👀👌👀👌👀👌👀 good shit go౦ԁ sHit👌 thats ✔ some good👌👌shit right👌👌th 👌 ere👌👌👌 right✔there ✔✔if i do ƽaү so my selｆ 💯 i say so 💯 thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 👌👌 👌НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ👌 👌👌 👌 💯 👌 👀 👀 👀 👌👌Good shitsign me the FUCK up 👌👀👌👀👌👀👌👀👌👀 good shit go౦ԁ sHit👌 thats ✔ some good👌👌shit right👌👌th 👌 ere👌👌👌 right✔there ✔✔if i do ƽaү so my selｆ 💯 i say so 💯 thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 👌👌 👌НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ👌 👌👌 👌 💯 👌 👀 👀 👀 👌👌Good shitsign me the FUCK up 👌👀👌👀👌👀👌👀👌👀 good shit go౦ԁ sHit👌 thats ✔ some good👌👌shit right👌👌th 👌 ere👌👌👌 right✔there ✔✔if i do ƽaү so my selｆ 💯 i say so 💯 thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 👌👌 👌НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ👌 👌👌 👌 💯 👌 👀 👀 👀 👌👌Good shit

**Comment by optimumtact** [comment]
Can't see any issues in the code from my cursory read

**Comment by LatD** [comment]
I'd also make the "transfer to beaker" button to be a switch (of 2 buttons next to each other for example, one of these would be active) since it's basically a switch and works differently than all the other real buttons on that machine.

**Comment by Incoming5643** [comment]
how bad are the load times on this?

**Comment by optimumtact** [comment]
It's time to upgrade from windows xp man

**Comment by Bobylein** [comment]
@ChangelingRain I would change it if I would know how

@LatD changed that, not sure if I like it but might be being used to the old one

@Incoming5643 Similiar to the load times it had before, I had both windows open for some time and there wasn't a real difference

**Comment by Bobylein** [comment]
also updated the album with the new gui pictures, so from my side it's ready.

**Comment by Bobylein** [comment]
so will something happen here or can I close it as there is no maintainer?

**Comment by PKPenguin321** [comment]
if it's ready either mergebeg in #coderbus or wait a long time, it'll get merged eventually

File: tgstation.dme [hunk 1]
```diff
@@ -9,7 +9,6 @@
 // END_FILE_DIR
 
 // BEGIN_PREFERENCES
-#define DEBUG
```
Comment (phil235) [inline tgstation.dme]: I don't think you intended to remove this.

**Comment by KorPhaeron** [comment]
@Bobylein

**Comment by optimumtact** [comment]
I am prepared to this one you fix the merge conflicts and readd the define

**Comment by Bobylein** [comment]
allright, just came back from a zombie apocalypse where I had no internet connection, will work on it.

File: tgstation.dme (hunk 1 above)
Comment (Bobylein) [inline tgstation.dme]: erhm yea... I switched debbuging in DreamMaker off in hope the lag would go away and didn't reenable it.

**Comment by optimumtact** [comment]
All ready to go?

**Comment by Bobylein** [comment]
Yea guess so

**Comment by optimumtact** [comment]
Thanks for your work

**Pull request merged**